backends/dpdk/dpdkArch.cpp

```diff
@@ -452,6 +452,18 @@ bool CollectMetadataHeaderInfo::preorder(const IR::P4Program *) {
 }
 
 bool CollectMetadataHeaderInfo::preorder(const IR::Type_Struct *s) {
+    /* Throw error if field width is greater than 64 bits */
```

Comment:
I think this pass is too early to have this check.
Later metadata fields are added a multiple places and those will not be handled by this check.
Can we have it either while converting to dpdk IR or as post codegen check?